cli/src/app.rs

```diff
@@ -259,11 +254,6 @@ pub fn add_subcommands(command: Command) -> Command {
                          system)",
                     ),
                     Arg::new("label").short('l').value_name("label"),
-                    Arg::new("verbose")
-                        .action(ArgAction::SetTrue)
-                        .short('v')
-                        .long("verbose")
```

Comment:
Removing the long form of this option for the analyze subcommand is a breaking change...at least for `phylum-ci`, which makes calls with this long form. Can the global verbosity option be updated to provide a long form? If not, then a release of the CLI with this breaking change will need to be coordinated with an update to `phylum-ci` to use the short form instead.